src/main/java/frc/team2412/robot/Subsystems/ClimbMotorSubsystem.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.RobotMap;
+
+public class ClimbMotorSubsystem extends SubsystemBase {
+	ClimbMotorSubsystem m_ClimbMotor;

```

Comment:
subsystem doesnt need an instance of itself

Reply:
Just removed it.